vllm/model_executor/layers/quantization/fp8.py

```diff
@@ -239,188 +241,6 @@ def apply(self,
             use_per_token_if_dynamic=False)
 
 
-class Fp8MoEMethod(FusedMoEMethodBase):
```

Comment:
I'd prefer to keep `Fp8MoEMethod` in `fp8.py` instead of creating another file. Can we just lazy import `fused_moe` like `UnquantizedFusedMoEMethod` (https://github.com/vllm-project/vllm/blob/main/vllm/model_executor/layers/fused_moe/layer.py#L91) does?

Reply:
yeah good point. i thought it was more complicated because we need to inherit from `FusedMoEMethodBase` but actually that base class doesn't involve any Triton import. have pushed the change